hedera-node/hedera-token-service-impl/src/main/java/com/hedera/node/app/service/token/impl/validators/CryptoTransferValidator.java

```diff
@@ -92,7 +92,7 @@ public void pureChecks(@NonNull final CryptoTransferTransactionBody op) throws P
                 validateTruePreCheck(acctAmount.hasAccountID(), INVALID_TRANSFER_ACCOUNT_ID);
                 uniqueTokenAcctIds.add(acctAmount.accountIDOrThrow());
                 netTokenBalance += acctAmount.amount();
-                if (!nonZeroFungibleValueFound && acctAmount.amount() != 0) {
+                if (!nonZeroFungibleValueFound && acctAmount.amount() >= 0) {
```

Comment:
@Neeharika-Sompalli shouldn't we allow a negative account amount here (i.e. `!= 0` is correct)? I thought we did allow negative amounts, but can't remember for sure